deployment/ansible/roles/cac-tripplanner.otp-data/defaults/main.yml

```diff
@@ -4,7 +4,5 @@
 #otp_data_dir: /var/otp
 otp_osm_source: https://s3.amazonaws.com/metro-extracts.mapzen.com/philadelphia_pennsylvania.osm.pbf
 otp_osm_filename: /var/otp/philadelphia.osm.pbf
-otp_gtfs_sources:
-  - septa_rail
-  - septa_bus
-  - patco
+otp_repo: "https://github.com/azavea/OpenTripPlanner.git"
```

Comment:
I'm not sure we should be using the azavea fork unless absolutely necessary.

`otp_version` can be a tag name, so we can just bring down v0.13 of the parent repository. If we do need to use the azavea fork in the future, we can trivially switch to using it then.